app.vue

```diff
@@ -2,7 +2,17 @@
   <Header />
   <Cart />
   <main class="min-h-[75vh] px-4">
-    <NuxtPage />
+    <div
+      v-if="
```

Comment:
This is good but making a note to move to computed property hasEnv() to clean up template